packages/block-library/src/button/editor.native.scss

```diff
@@ -25,7 +25,11 @@
 	padding: $block-spacing;
 	max-width: 580px;
 	min-width: 108px;
-	margin: $panel-padding;
+	margin: $panel-padding 2 * $panel-padding;
+}
+
+.buttonsSelected {
+	margin: $panel-padding + $border-width + $block-selected-child-margin + $block-selected-child-to-content;
```

Comment:
Just a note again. This one probably also will be removed in Piotr approach on bordering